mne/preprocessing/_csd.py

```diff
@@ -288,9 +287,7 @@ def compute_bridged_electrodes(inst, lm_cutoff=16, epoch_threshold=0.5,
 
     # kernel density estimation
     kde = gaussian_kde(ed_flat[ed_flat < lm_cutoff])
-    with warnings.catch_warnings():
-        warnings.filterwarnings(
-            'ignore', 'invalid value encountered in true_divide')
+    with np.errstate(invalid='ignore'):
```

Comment:
FYI @alexrockhill this is the more standard way to ignore numerical warnings in NumPy. FWIW in this case it mattered because the error message said `in divide` rather than `in true_divide`

Reply:
Good to know, thanks